cmake/GlooConfig.cmake.in

```diff
@@ -0,0 +1,32 @@
+# - Config file for the Gloo package
+# It defines the following variables
+#  GLOO_INCLUDE_DIRS       - include directories for Gloo
+#  GLOO_LIBRARIES          - libraries to link against
+#  GLOO_BUILDER_LIBRARIES  - builder libraries to link against
+#  GLOO_CUDA_LIBRARIES     - cuda libraries to link against
+
+# library version information
+
+set(GLOO_VERSION_MAJOR @GLOO_VERSION_MAJOR@)
+set(GLOO_VERSION_MINOR @GLOO_VERSION_MINOR@)
+set(GLOO_VERSION_PATCH @GLOO_VERSION_PATCH@)
+set(GLOO_VERSION "@GLOO_VERSION@")
+
+# import targets
+include ("${CMAKE_CURRENT_LIST_DIR}/GlooTargets.cmake")
+
+# include directory
+#
+# Newer versions of CMake set the INTERFACE_INCLUDE_DIRECTORIES property
+# of the imported targets. It is hence not necessary to add this path
+# manually to the include search path for targets which link to gflags.
```

Comment:
I don't understand this comment (perhaps copied from Caffe2?). It says it is not necessary to add the include path manually, but this is exactly what happens below (setting GLOO_INCLUDE_DIRS).

Reply:
Yeah, this is copied from gflags - it is kind of like a backward compatibility setting so that if the dependent libraries choose to, they should still be able to use the old-fashioned variable names.